<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitForSignalWidget</class>
 <widget class="QWidget" name="WaitForSignalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title">Edit signal to wait for</string>
  </property>
  <property name="whatsThis">
   <string comment="@info:whatsthis">&lt;para&gt;Set the emitter name and the signal to wait for.&lt;/para&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="WaitForSignalVerticalLayout">
   <item>
    <widget class="QGroupBox" name="waitForSignalGroupBox">
     <property name="title">
      <string comment="@title:group">Wait for signal</string>
     </property>
     <layout class="QHBoxLayout" name="waitForSignalGroupBoxHorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="labelVerticalLayout">
        <item>
         <widget class="QLabel" name="emitterNameLabel">
          <property name="text">
           <string comment="@label:textbox">Emitter name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>emitterNameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="signalNameLabel">
          <property name="text">
           <string comment="@label:textbox">Signal name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>signalNameLineEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="valueVerticalLayout">
        <item>
         <widget class="KLineEdit" name="emitterNameLineEdit">
          <property name="whatsThis">
           <string comment="@info:whatsthis">&lt;para&gt;The name of the QObject that emits the signal.&lt;/para&gt;
&lt;para&gt;Note that the name is not the class of the object, but the string returned by its objectName() method.&lt;/para&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="signalNameLineEdit">
          <property name="whatsThis">
           <string comment="@info:whatsthis">&lt;para&gt;The name of the signal.&lt;/para&gt;
&lt;para&gt;The name must be written as &lt;em&gt;signalName(TypeOfFirstArgument, TypeOfSecondArgument, TypeOfThirdArgument...)&lt;/em&gt;, without including &lt;em&gt;SIGNAL()&lt;/em&gt; wrapping text.&lt;/para&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="waitForSignalWidgetSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
